cla_backend/apps/checker/views.py

```diff
@@ -134,3 +135,25 @@ def post_save(self, obj, created=False):
                 if form.is_valid():
                     form.save(obj.created_by)
                     notify_callback_created(obj)
+
+
+class DiagnosisViewSet(
+    PublicAPIViewSetMixin,
+    DiagnosisModelMixin,
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
```

Comment:
does this mean anyone can get back a diagnosis if they know the id? maybe we can lock it based on a session?